### PR TITLE
Add Farquhar Parameters

### PR DESCRIPTION
**Pull request by nefrathenrici**
Also removes an errant `alias` left in the autotrophic respiration parameters

### CODE REVIEW

File: src/parameters.toml [hunk 1]
```diff
@@ -38,6 +37,93 @@ value = 0.25
...
+type = "float"
+description = "Intercelluar O2 concentration (mol/mol); taken to be constant"
+
+[photosystem_II_quantum_yield]
+value = 0.6
+type = "float"
+description = "Quantum yield of photosystem II (Bernacchi, 2003; unitless)"
+
+[Farquhar_curvature_parameter]
+value = 0.6
```
Comment (kmdeck) [inline src/parameters.toml:104]: I believe we are using 0.9 for this

File: src/parameters.toml [hunk 2]
```diff
@@ -38,6 +37,93 @@ value = 0.25
...
+type = "float"
+description = "Reference temperature equal to 25 degrees Celsius (K)"
+
+[intercellular_CO2_concentration]
+value = 0.209
+type = "float"
+description = "Intercelluar O2 concentration (mol/mol); taken to be constant"
+
+[photosystem_II_quantum_yield]
+value = 0.6
```
Comment (kmdeck) [inline src/parameters.toml:99]: Let's use Φ = 0.7 and cite Bonan (2019)

File: src/parameters.toml (hunk 1 above)
Comment (kmdeck) [reply src/parameters.toml:104]: We can cite (von Caemmerer 2000,2013; Bernacchi et al.2003,2013; von Caemmerer et al. 2009).

File: src/parameters.toml [hunk 3]
```diff
@@ -38,6 +37,93 @@ value = 0.25
...
+type = "float"
+description = "Energy of activation for CO2 (J/mol)"
+
+[O2_activation_energy]
+value = 36380
+type = "float"
+description = "Energy of activation for oxygen (J/mol)"
+
+[Vcmax_activation_energy]
+value = 58520
```
Comment (kmdeck) [inline src/parameters.toml:69]: 65330 (I had read the Table incorrectly when inputting this originally)

File: src/parameters.toml [hunk 4]
```diff
@@ -38,6 +37,93 @@ value = 0.25
...
+type = "float"
+description = "Energy of activation for Vcmax (J/mol)"
+
+["Γstar_activation_energy"]
+value = 58520
```
Comment (kmdeck) [inline src/parameters.toml:74]: 37830

File: src/parameters.toml [hunk 5]
```diff
@@ -38,6 +37,93 @@ value = 0.25
...
+value = 4.049e-4
+type = "float"
+description = "Michaelis-Menten parameter for CO2 at 25 °C (mol/mol)"
+
+[O2_michaelis_menten]
+value = 0.2874
+type = "float"
+description = "Michaelis-Menten parameter for O2 at 25 °C (mol/mol)"
+
+[CO2_activation_energy]
```
Comment (kmdeck) [inline src/parameters.toml:58]: For all activation energy numbers, can we site Bonan (2019), Table 11.2 and Bernacchi et al. (2001, 2003)?

File: src/parameters.toml [hunk 6]
```diff
@@ -38,6 +37,93 @@ value = 0.25
...
+type = "float"
+description = "Γstar at 25 °C (mol/mol)"
+
+[CO2_michaelis_menten]
+value = 4.049e-4
+type = "float"
+description = "Michaelis-Menten parameter for CO2 at 25 °C (mol/mol)"
+
+[O2_michaelis_menten]
+value = 0.2874
```
Comment (kmdeck) [inline src/parameters.toml:54]: 0.2784

File: src/parameters.toml [hunk 7]
```diff
@@ -38,6 +37,93 @@ value = 0.25
 type = "float"
 description = "Factor of relative contribution or Rgrowth (-)"
 
+
+# Farquhar Parameters
+
+[CO2_compensation_point_25c]
+value = 4.275e-5
```
Comment (kmdeck) [inline src/parameters.toml:44]: For these 25C numbers, we can cite Bernacchi et al. (2001)

File: src/parameters.toml [hunk 8]
```diff
@@ -38,6 +37,93 @@ value = 0.25
...
+value = 46390
+type = "float"
+description = "Energy of activation for Rd (J/mol)"
+
+[kelvin_25C]
+value = 298.15
+type = "float"
+description = "Reference temperature equal to 25 degrees Celsius (K)"
+
+[intercellular_CO2_concentration]
```
Comment (kmdeck) [inline src/parameters.toml:93]: long name does not match description, typo in description.

should be O2

**Review by kmdeck — APPROVED** [review]
looks good after changes suggested, thanks!

File: src/parameters.toml (hunk 2 above)
Comment (nefrathenrici) [reply src/parameters.toml:99]: Bonan and Bernacchi, 2003 or just Bonan?

Comment (kmdeck) [reply src/parameters.toml:99]: both, thank you!

**Pull request merged**